docker/install/ubuntu_install_papi.sh

```diff
@@ -26,7 +26,7 @@ apt-get update --fix-missing
 apt-get install -y linux-tools-common linux-tools-generic
 
 cd /
-git clone https://bitbucket.org/icl/papi.git
+git clone -b stable-6.0 --single-branch https://bitbucket.org/icl/papi.git
```

Comment:
This will not work as we need to include this commit (https://bitbucket.org/icl/papi/commits/a0d7cf0d5929f2115636e77317daa0f00d075e08) in order for papi to work with CUDA.

Reply:
i see. well, this is quite inconvenient at the moment. I think this error can be fixed by solving the papi compile issue
in the papi repo, that is another solution, but i am not sure how long that will take them to fix.
we could alternatively checkout the commit you refernced above instead of the stable tag (as a temporary fix)?
that  would get the docker ci_cpu image build to succeed at least.

Reply:
i pointed the checkout to the latest commit in papi where the build succeeds
https://github.com/mherkazandjian/tvm/commit/eb6e7d235ebd1f41bcf44ee7c7b306a5911930c8

Reply:
papi just reverted the offending commit
https://bitbucket.org/icl/papi/issues/84/build-of-the-master-branch-is-failing-with
we can close close this PR and the issue https://github.com/apache/tvm/issues/8173